Add MLP subsection in related works (#8)
chhoumann/P6: sections/related-works.tex

Another approach to time series forecasting is the use of neural networks. A popular model is to use recurrent neural networks and derivations of this model. Recurrent neural networks are similar to feedforward networks, but have memory of the past which is achieved through a hidden state which is passed recurrently through from previous layer as input to the succeeding layer. This makes it capable of learning short patterns. 
A primary issue with RNNs are the vanishing gradient and the exploding gradient problem with longer patterns, which more powerful models seek to improve upon. \cite{AIModernApproach}\cite{hands-onML}

\subsection{MLP models}
A Multilayer Perceptron consists of an input and output layer as well as one or more hidden layers. These hidden layers are made up of multiple neurons. 

> Since MLPs are feedforward algorithms the inputs are combined with an initial weight in a weight sum and then given to the activation function. 

Reviewer: ```suggestion
Since MLPs are feedforward algorithms, the inputs are combined with an initial weight in a weighted sum and then passed to the activation function. 
```

> Each of these hidden layers are then feeding their output and their internal representation of the data to the next layer in the line until we reach the output layer.\cite{bentoMultilayerPerceptronExplained2021}

Reviewer: ```suggestion
Each of these hidden layers then feeds their output and their internal representation of the data to the next layer in the network until the output layer is reached.\cite{bentoMultilayerPerceptronExplained2021}
```
